tests/manage/pv_services/pv_encryption/test_pvc_with_tc_overrie_vn.py

```diff
@@ -0,0 +1,116 @@
+import logging
+import pytest
+
+from ocs_ci.ocs import constants
+from ocs_ci.framework import config
+from ocs_ci.framework.testlib import (
+    skipif_ocs_version,
+    ManageTest,
+    tier1,
+)
+
+from ocs_ci.helpers.helpers import (
+    create_unique_resource_name,
+)
+
+
+log = logging.getLogger(__name__)
+
+kmsprovider = constants.VAULT_KMS_PROVIDER
+argnames = ["kv_version", "kms_provider", "use_vault_namespace"]
```

Comment:
It would be better to add `vaulttenantsa` method as well. Since this feature is expected to work for both vaulttoken and vaulttenantsa methods.

Reply:
Will raise separate PR for this